src/extension.ts

```diff
@@ -16,10 +16,11 @@ export function activate(context: vscode.ExtensionContext) {
             const config = vscode.workspace.getConfiguration('createReactComponent');
             const isTypescript = config.get('language') === 'typescript';
 
-            const options = {
+            const options: TemplateOptions = {
                 name: componentName,
                 testLibrary: <TestLibrary>config.get('testingLibrary'),
                 cleanup: <boolean>config.get('testingLibrary.cleanup'),
+                functionType: <FunctionType>config.get('functionType')
```

Comment:
Some changes came in just before you created this PR. I have changed the type casts here to use the `as` syntax. You'll have to rebase 😬